app/controllers/infra_networking_controller.rb

```diff
@@ -478,6 +487,7 @@ def action_type(type, amount)
   def show_details(db, options = {}) # Pass in the db, parent vm is in @vm
     association = options[:association]
     conditions  = options[:conditions]
+    clickable   = options[:clickable].nil? || options[:clickable].nil?
```

Comment:
@skateman before/after || are same

Reply:
oh my, how this could happen :rofl: thanks :beers: